### ISSUE TITLE
Ghost/Phantom names

### ISSUE THREAD

**Issue by tuurma** [eXistSolutions/LGPN-ling#660]
Dear Sophie,

I've been preparing the conversion for ghost names but, upon consideration, I would like to remove the funeral cross from the name and use the checkbox, exactly like proposed for "bilingual".

![image](https://user-images.githubusercontent.com/449468/150965225-69d9a423-1cfc-42b3-a5a1-a83e081a5a79.png)

This way we would be able to easily search for these names, whereas searching for special characters like the cross is unnecessarily difficult and it is not really proper approach in terms of data modelling.

Below you can also see another problem with automated conversion of ghost names. I converted the `†Πνυτόκλα` into `Πνυτόκλα†` but the name id would become `Pnytokla` which already exists. I was under impression that ghost names cannot exist as "regular" names. Is it just an editor mistake for Pnytokla?

![image](https://user-images.githubusercontent.com/449468/150966267-d7c27f84-f28b-40a2-b4bf-d695d0c95ae8.png)

**Comment by tuurma** [comment]
I already added the checkboxes in the new editing form and also added corresponding facets. Doing this, I noticed we do not have facets for GPR, fictitious and gender, so tentatively added these as well.

![image](https://user-images.githubusercontent.com/449468/150997376-0e232ee5-37e9-4109-8924-d39672618b21.png)

**Comment by Hippodamie** [comment]
Dear Magdalena,

many thanks for all what you did! Perfect and thank you too for the 2 new boxes. For Pnytokla, it was a mistake: we forgot to delete the ancient one (without cross), after we added the cross (it created 2 entries when we added the cross to the ancient one). I now deleted it. It can happen there are still the 2 for some other , for the same reason. Sorry if it happens again

About now the facets you added:
we would like to have Chronology coming first of all;
then Dialect-aréal;
-then GPR (visible only by the arrow).
-bilingual labels have to get out of Dialect-areal facet and come under GPR, with the arrow too.
-fictitious (by the arrow); french: fictionnels
-Gender
-Lexicon etc. like it is;
-headedness  and then Abbreviations: just after Classification (both by the arrow);
- then Genetics, as it is;
-ghost names (it is the good vocabulary)  (by the arrow)
-dernières modifications: at the end (arrow too).

N.B.: headedness; in French, call it just: X vs Y
Shortened/truncated names: French: Noms raccourcis et tronqués
